hdKeysGenerator/go.mod

```diff
@@ -0,0 +1,50 @@
+module github.com/ElrondNetwork/elrond-tools-go/hdKeysGenerator
+
+go 1.18
+
+require (
+	github.com/ElrondNetwork/elrond-go v1.3.35
+	github.com/ElrondNetwork/elrond-go-core v1.1.16-0.20220622092812-cc07c736c3b8
+	github.com/ElrondNetwork/elrond-go-logger v1.0.7
+	github.com/ElrondNetwork/elrond-sdk-erdgo v1.1.0
+	github.com/stretchr/testify v1.7.1
+	github.com/urfave/cli v1.22.9
+	golang.org/x/sync v0.0.0-20210220032951-036812b2e83c
+)
+
+require (
```

Comment:
can we have only one require list? 
Why do we need 2 batches of arwen replaces?